.github/Pull_Request_Template.md

```diff
@@ -0,0 +1,31 @@
+# Problem Statement
+
+_Describe the problem this patch intends to solve._
+
+## Design
+
+- _For Bug describe the fix here._
+- _For feature, post the link to the solution page._
+
+## Coding
+
+- _Coding conventions are followed and code is consistent. Yes/No?_
+- _Confirm All CODACY errors are resolved. Yes/No?_
+
+## Testing
+
+- _Confirm that Test Cases are added (for both the cases, fix and feature). Yes/No?_
+- _Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability. Yes/No?_
+- _Confirm Testing was performed with installed RPM. Yes/No?_
+
+## Checklist
+
+- _PR is self-reviewed? Yes/No?_
+- _Any interface change Yes/No?_
+- _Change notified to other gatekeepers: Yes/No?_
+- _Code reviews done and addressed: Yes/No?_
+- _Codacy issues reviewed and addressed: Yes/No?_
+- _Deployment test : Done/Not?_
+- _UT/smoke test: Done/Not?_
+- _Jira number added to PR: Yes/No?_
+- _Github merge done using UI: Yes/No?_
```

Comment:
1)side effects on other features - deployment/update/node-replacement
2)Changes to quick-start-guide/community impact

Reply:
@TechWriter-Mayur, please add further points as suggested?